File: pbwebwcl/src/app/rendicontazione/commons/requests/salva-qualifiche-request.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

import { QualificaFormDTO } from "../dto/qualifica-form-dto";

export class SalvaQualificheRequest {
    constructor(
        public listaQualificaFormDTO: Array<QualificaFormDTO>
    ) { }
}

File: pbweb/pbweb-jar/src/main/java/it/csi/pbandi/pbweb/integration/dao/request/SalvaQualificheRequest.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbweb.integration.dao.request;

import java.util.ArrayList;

import it.csi.pbandi.pbweb.dto.QualificaFormDTO;

public class SalvaQualificheRequest {
	
	private ArrayList<QualificaFormDTO> listaQualificaFormDTO;

	public ArrayList<QualificaFormDTO> getListaQualificaFormDTO() {
		return listaQualificaFormDTO;
	}

	public void setListaQualificaFormDTO(ArrayList<QualificaFormDTO> listaQualificaFormDTO) {
		this.listaQualificaFormDTO = listaQualificaFormDTO;
	}

}
